README.md

```diff
@@ -41,9 +41,15 @@ Review the [Create a Block tutorial](/docs/getting-started/create-block/README.m
 
 ### Contribute to Gutenberg
 
-Gutenberg is an open-source project and welcomes all contributors from code to design, from documentation to triage. The project is built by many contributors and volunteers and we'd love your help building it.
+Gutenberg is an open-source project and welcomes all contributors from code to design, and from documentation to triage. The project is built by many contributors and volunteers and we'd love your help building it.
 
-See the [Contributors Handbook](https://developer.wordpress.org/block-editor/contributors/) for all the details on how you can contribute. See [CONTRIBUTING.md](https://github.com/WordPress/gutenberg/blob/HEAD/CONTRIBUTING.md) for the contributing guidelines.
+See the [Contributors Handbook](https://developer.wordpress.org/block-editor/contributors/) for all the details on how you can contribute. 
+
+To get up and running quickly with **code contribution** see [Getting Started With Code Contribution](/docs/contributors/code/getting-started-with-code-contribution.md). Also check out the other resources available on the [Code Contributions](/docs/contributors/code/README.md) page.
+
+To get started contributing to **design** see [Design Contributions](/docs/contributors/design/README.md). If you want to work on **documentation** head on over to the [Documentation Contributions](/docs/contributors/documentation/README.md) page, and if you want to help out with **triaging the repository** then the [Get involved in triage](/docs/contributors/triage.md) page is the place to go.
```

Comment:
If the goal of this PR is to fix

> The route to get to the Getting Started page isn't easily discoverable especially for newcomers to code contribution.

I wouldn't really add this info in this README as it's already available from https://developer.wordpress.org/block-editor/contributors/

I think the goal of this PR is to make contribution to code a bit easier by adding some direct links in this README

Reply:
I was trying to avoid placing undue emphasis on code contributions, and trying to include and encourage contributions of all kinds.

That said, the most likely people to read the README file in the repo are developers - but I think it's fair to include all contributors in the README and provide links to make their path to contributing easier.

Happy to remove if you insist that it shouldn't be there.

Reply:
> the most likely people to read the README file in the repo are developers

That's exactly the point of this action

There are enough paths for other type of contributors but I think the point of this action is to simplify contributions of the most likely to contribute profiles 

I would remove this paragraph because this info it's already available in https://developer.wordpress.org/block-editor/contributors/ and in https://github.com/WordPress/gutenberg/blob/trunk/CONTRIBUTING.md

Reply:
Okay, done!